crates/ruff_linter/src/rules/flake8_pytest_style/rules/helpers.rs

```diff
@@ -45,6 +48,66 @@ pub(super) fn is_pytest_parametrize(call: &ExprCall, semantic: &SemanticModel) -
             matches!(qualified_name.segments(), ["pytest", "mark", "parametrize"])
         })
 }
+/// Whether `path` likely leads to a Pytest test file.
+///
+/// Pytest determines a test file using [a number of conventions and settings][1].
+/// Only the conventions are reflected here.
+/// This function returns true if the file name (`.py` notwithstanding):
+///
+/// * Starts with `test_`, or
+/// * Ends with `_test`
+///
+/// It is thus prone to both false positives and false negatives.
+///
+/// [1]: https://docs.pytest.org/en/stable/explanation/goodpractices.html#conventions-for-python-test-discovery
+#[allow(dead_code)]
+pub(crate) fn is_likely_pytest_test_file(path: &Path) -> bool {
```

Comment:
We should delete this function if it isn't used anywhere other than in a docstring.